docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/api/index.md

```diff
@@ -0,0 +1,144 @@
+
+The endpoints focus on the main operations in the workflow around:
+
+1. Retrieving existing data products and their event specifications, also known as tracking scenarios in the current v1 API version
+2. Creating or editing new or existing data products
+3. Seeing the data product history
+4. Managing user subscriptions
+
+Each request will need to include your company's `organizationID` which is a UUID that can be retrieved from the URL immediately following the .com when visiting console:
+
+![](images/orgID.png)
+
+```mdx-code-block
+import GetConsoleApiKey from "@site/docs/reusable/get-console-api-key/_index.md"
+
+<GetConsoleApiKey/>
+```
+
+## Retrieving Information about Data Products
+
+The following `GET` requests are designed to allow you to access information about data products. These data products could be stored in either your development environment registry or your production environment registry.
+
+### Retrieve a List of All Data Products
+
+To retrieve a comprehensive list of all data products in your organization, you can use the following GET request:
+
+`**GET** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1`
+
+Query parameter `organizationId` is required.
+
+### Retrieving Information about a Specific Data Product
+
+`**GET** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1/{dataProductId}`
+
+Query parameters `organizationId` and `dataProductId` are required.
+
+### Retrieve History Information for a Data Product
+
+If you wish to retrieve information about a specific data product, you can use the following GET request:
+
+`**GET** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1/{dataProductId}/history`
+
+You can pass several parameters to control the result of the response:
+
+- **before**: returns records equal or less than the timestamp in the ISO-8601 format
+- **limit**: limits the number of records
+- **offset**: skip the first N results
+- **order**: order of returned records, `asc` or `desc`. Defaults to `desc`.
+
+Query parameter `organizationId` is required.
+
+## Creating and updating Data Products
+
+### Creating a Data Product
+
+This `POST` request allows you to create a new data product within an organization.
+
+`**POST** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1`
+
+The request body is mandatory and should be in JSON format. Here’s an example of how the request body might look:
+
+```json
+{
+  "name": "Performance tracking",
+  "description": "Tracks performance",
+  "domain": "Marketing",
+  "owner": "IT department",
+  "accessInstructions": "The data can be accessed through company Superset"
+}
+```
+
+### Updating a Data Product
+
+Use this request to update a data product. The `dataProductId` is required, along with a valid request body.
```

Comment:
The minimum payload on update would be the same as on creation but with the addition of the required `status` field. On creation, by default, will set the status to `draft`. Don't know if it is worth mentioning it.

Reply:
done - I think it is useful I added it thanks